### PR TITLE
Remove AltTester and add it as a submodule

### LINKED ISSUE #3: Please remove AltTester and add it as a submodule, this will make it easier in the future to update the example
Please remove AltTester and add it as a submodule, this will make it easier in the future to update the example

_Originally posted by @RobertPoienar in https://github.com/alttester/EXAMPLES-NewInputSystem--CoinCollector-CSharp/pull/2#pullrequestreview-1182018825_

### PR DESCRIPTION
**Pull request by andreion1ca**
AltTester was removed and added as a submodule to make it easier in the future to update the example

### CODE REVIEW

**Comment by andreion1ca** [comment]
@RobertPoienar @robert-altom @ReichEdvin @gomboskriszta15 @iulianatodoran can one of you have a look at this PR?

**Review by RobertPoienar — APPROVED** [review]
LGTM

**Comment by iulianatodoran** [comment]
- As discussed earlier this month, it would be helpful to document in the `README.md` file the steps if we use a submodule, so the users have clear info on what to do to get the AltTester submodule in the project. 
- Also, there was an idea to provide a link for users to download the AltTester package with instructions for importing if they prefer this method, and not to use the submodule. 

So, **in conclusion**, the `README.md` file should contain both ways of using AltTester in the project
- add the submodule and use the master branch
- link for download AltTester + import in Unity

File: README.md [hunk 1]
```diff
@@ -3,8 +3,20 @@
 This project contains C# AltTester tests for a project using the New Input System.
 The tested actions are: tap object, key down/ key up, press key, click object, scroll, begin/end touch and tap simulated with swipe.
 
+## Before running the tests
+To run the tests, you must include the AltTester Unity SDK in the project. To do that, you can choose between the following ways:
+1. Add the AltTester Unity SDK submodule to the project
+    - in the <i> New Input System proj/Assets </i> folder add the submodule using the <i><b> git submodule add https://github.com/alttester/AltTester-Unity-SDK.git </b></i> command;
```
Comment (iulianatodoran) [inline README.md]: For commands, you could use  `` at the beginning and the end of the command to look like a command
This applies to all commands in this file

File: README.md [hunk 2]
```diff
@@ -3,8 +3,20 @@
...
+    - make sure that the submodule added is on the master branch (you can use the following command <i><b> cd AltTester-Unity-SDK && git checkout master</b></i>).
+
+    <br> 
+2. Download AltTester Unity SDK and import it into Unity 
+    - download the AltTester Unity SDK from the Altom website (https://altom.com/testing-tools/alttester/) or using this link https://altom.com/app/uploads/AltTester/sdks/AltTester.unitypackage;
+    - open the project in Unity and from the menu, go to <i> Assets -> Import Packages -> Custom Package... </i> and select the AltTester Unity Package downloaded before;
```
Comment (iulianatodoran) [inline README.md]: easier way is to drag-n-drop directly in the Assets folder

File: README.md [hunk 3]
```diff
@@ -3,8 +3,20 @@
 This project contains C# AltTester tests for a project using the New Input System.
 The tested actions are: tap object, key down/ key up, press key, click object, scroll, begin/end touch and tap simulated with swipe.
 
+## Before running the tests
+To run the tests, you must include the AltTester Unity SDK in the project. To do that, you can choose between the following ways:
+1. Add the AltTester Unity SDK submodule to the project
+    - in the <i> New Input System proj/Assets </i> folder add the submodule using the <i><b> git submodule add https://github.com/alttester/AltTester-Unity-SDK.git </b></i> command;
+    - make sure that the submodule added is on the master branch (you can use the following command <i><b> cd AltTester-Unity-SDK && git checkout master</b></i>).
```
Comment (iulianatodoran) [inline README.md]: When a release is made for AltTester the user has to make a git pull on the master branch, maybe this info is useful too if they already have the project and thus are aware that the AltTester version was upgraded

**Review by RobertPoienar — CHANGES_REQUESTED** [review]
.gitmodules shouldn't be empty, it seems you didn't added correctly the submodule

File: README.md [hunk 4]
```diff
@@ -3,8 +3,21 @@
...
+1. Add the AltTester Unity SDK submodule to the project
+    - in the <i> New Input System proj/Assets </i> folder add the submodule using the `` git submodule add https://github.com/alttester/AltTester-Unity-SDK.git `` command;
+    - make sure that the submodule added is on the master branch (you can use the following command `` cd AltTester-Unity-SDK && git checkout master``);
+    - also, if you already have the project, you should make a ``git pull`` on the master branch, in order to ensure that you are using the latest version of AltTester.
+
+    <br> 
+2. Download AltTester Unity SDK and import it into Unity 
+    - download the AltTester Unity SDK from the Altom website (https://altom.com/testing-tools/alttester/) or using this link https://altom.com/app/uploads/AltTester/sdks/AltTester.unitypackage;
+    - open the project in Unity and from the menu, go to <i> Assets -> Import Packages -> Custom Package... </i> and select the AltTester Unity Package downloaded before or just drag-n-drop the package in the Assets folder;
+    - a pop-up will appear, select All and click on Import.
```
Comment (iulianatodoran) [inline README.md:17]: this step seems a bit too much information to give, the users in unity usually know how to import packages so the previous step should be enough.

**Pull request merged**